lambdas/package/index.py

```diff
@@ -0,0 +1,144 @@
+"""
+Provide the head of a (potentially gzipped) file in S3. Stream to limit
```

Comment:
this comment should be adjusted probably

Reply:
Prefixes and objects are now returned separately in the folder/prefix view.